volatility3/framework/plugins/linux/check_afinfo.py

```diff
@@ -51,10 +51,24 @@ def _check_members(self, var_ops, var_name, members):
                 yield check, addr
 
     def _check_afinfo(self, var_name, var, op_members, seq_members):
-        for hooked_member, hook_address in self._check_members(
-            var.seq_fops, var_name, op_members
-        ):
-            yield var_name, hooked_member, hook_address
+        # check if object has a least one of the members used for analysis by this function
+        required_members = ["seq_fops", "seq_ops", "seq_show"]
+        for member in required_members:
```

Comment:
Hmmm, I know it's not a big deal, but this is traversing all the elements do output the debug information, and then does it again (effectively) as part of the any operator.  The debug information will probably only be checked if they're all false, so it might not be useful at all?

Reply:
Yes, I agree with you. Not the best. 

I wanted to have some difference between "The plugin run and there was no output, because there was nothing hooked" and "The plugin run and there was no output, because the kernel version isn't supported". 

The later might lead to a false sense of security. I'll work up a better way to show that difference.

Reply:
I've tweaked this now, but might be abusing the `PluginRequirementException` - let me know what you think.